src/core/src/InfiniteDimensionalMCMCSampler.C

```diff
@@ -27,7 +27,6 @@
 
 #include <cmath>
 #include <algorithm>
```

Comment:
Can you add `#include <sstream>` here?  Getting a compiler error regarding incomplete `stringstream` types below.